CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
    * ADDED: steps maneuver improvements [#4960](https://github.com/valhalla/valhalla/pull/4960)
    * ADDED: instruction improvements for node-based elevators [#4988](https://github.com/valhalla/valhalla/pull/4988)
    * ADDED: increased precision in route lengths [#5020](https://github.com/valhalla/valhalla/pull/5020)
+   * ADDED: `ignore_constructions` allows routing on ways with construction tag [#5030](https://github.com/valhalla/valhalla/pull/5030)
```

Comment:
drop the plural everywhere please (sorry i know english is weird)

```suggestion
   * ADDED: `ignore_construction` allows routing on ways with construction tag [#5030](https://github.com/valhalla/valhalla/pull/5030)
```

Reply:
Done!